### DISCUSSION TITLE
Are `.theme` files supported?

### DISCUSSION THREAD

**Discussion by beedell-roke** [nptr/msstyleEditor#119] [Q&A] [answered]
1. ![image](https://user-images.githubusercontent.com/42837531/235324620-c10f86e7-b7cf-4798-b875-a1196ede4d84.png)
2. ![image](https://user-images.githubusercontent.com/42837531/235324632-366254c7-f4f1-454f-8c3c-139cdfbb7dd7.png)
3. ![image](https://user-images.githubusercontent.com/42837531/235324590-5453239d-1c0a-40fd-a50b-49631f730892.png)

I created this one – [`(#000000).txt.theme`](https://github.com/nptr/msstyleEditor/files/11360095/000000.txt) – in Windows 10 or 11 in the last 6 months, so it should, surely?

**Comment by nptr** [comment, marked as answer]
No, they are not supported, but there is not much in them anyways. And since they are just text files you can modify them with any text editor.

The message you are seeing is misleading here. It refers to the Windows XP styles, which also had a .msstyles extension but are incompatible now. I always display this message when a file that is not real .msstyles file fails to load.
